boot.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/** @var rex_addon $this */
+
+if (!rex::isBackend() || !rex::getUser()) {
+    return;
+}
+
+rex_view::addCssFile($this->getAssetsUrl('ydeploy.css'));
+
+rex_extension::register('PAGE_BODY_ATTR', 'rex_ydeploy_handler::addBodyClasses');
```

Comment:
Willst du die Classes noch in einem anderen Kontext nutzen? Ansonsten könnten diese auch direkt am Badge notiert werden.

Reply:
Mein Gedanke war schon, dass man die dann auch noch für andere Zwecke nutzen kann. Weitere Dinge über css ein-/ausblenden, Farbanpassungen je nach Stage, oder ähnliches. 
So ganz konkrete Use cases habe ich also nicht, aber dachte mir, vielleicht kann man die Klassen mal noch gebrauchen.